src/TriggerCandidateMakerHorizontalMuon.cpp

```diff
@@ -87,15 +90,14 @@ TriggerCandidateMakerHorizontalMuon::operator()(const TriggerActivity& activity,
 
   // // If it is not, move the window along.
   // else {
-  //   // TLOG_DEBUG(TRACE_NAME) << "TAWindow is at required length but specified threshold not met, shifting window along.";
-  //   m_current_window.move(activity, m_window_length);
+  //   // TLOG_DEBUG(TRACE_NAME) << "TAWindow is at required length but specified threshold not met, shifting window
+  //   along."; m_current_window.move(activity, m_window_length);
```

Comment:
I have no idea what happened here, but clang formatting is wrong here.
I would just remove this entire block? Or:

```suggestion
  //   // TLOG_DEBUG(TRACE_NAME) << "TAWindow is at required length but specified threshold not met."
  //   //                        << "shifting window along.";
  //   m_current_window.move(activity, m_window_length);
```